openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py

```diff
@@ -1,6 +1,10 @@
+from pathlib import Path
+from typing import Any, Dict, Type
```

Comment:
In general I don't think annotating tests is worthwhile - although annotating the helper functions like `extract_info_from_lammps_atom_lines` is fine

Reply:
I've removed the Dict and Type imports in [this commit](https://github.com/openforcefield/openff-interchange/commit/424de3ba75509b5b290cad53d741deacfe6d0ab6).The pathlib import is necessary to fetch the current working directly (and because I prefer Path objects over strings :P), and the typing.Any import is to annotate `extract_info_from_lammps_atom_lines()`.

Reply:
Good - highlighting the `pathlib` import was incidental on my part